Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -318,6 +319,65 @@ extension _RSA.Signing {
     }
 }
 
+extension _RSA.Encryption {
+    public typealias PublicKey = _RSA.Signing.PublicKey
+    public typealias PrivateKey = _RSA.Signing.PrivateKey
```

Comment:
Typealiases are just a bit too weak for use in this case. Can I suggest we add new types that just so happen to hold the Signing key types?

Reply:
Hm... in that case, how would you feel about moving both key types into the `_RSA` "namespace" so that they're specific to neither subset (given that they are in fact nonspecific in terms of both implementation and usage).

Reply:
It's still preferable to domain-separate the keys. While the key itself is not specific, using a key simultaneously for different purposes is almost always a red flag in a cryptosystem.